src/Butil/Bit.Butil.Demo/Server/wwwroot/css/app.css

```
@@ -1,886 +1,2252 @@
+    overflow: auto;
+    direction: ltr;
+    flex-flow: column;
+    position: relative;
+    line-height: var(--demo-code-line-height);
+    font-size: var(--demo-code-font-size);
+    /* Exactly one line's worth, derived from the same metrics rather than rounded to a nice
+       number so it lands on the height a single-line pane takes naturally: an empty output pane
+       then matches a one-line one instead of collapsing to a hairline, and the copy button -
+       which is taller than a line of code - sits dead centre in it rather than a fraction shy. */
+    min-height: calc(2 * var(--bit-shp-brd-width) + 2 * var(--demo-code-pad)
+                   + var(--demo-code-font-size) * var(--demo-code-line-height));
```

Stylelint (17.14.0)

[error] Unexpected newline before "+" (scss/operator-no-newline-before)

(scss/operator-no-newline-before)